docs/system-admin-guide/manage-work-packages/work-package-status/README.md

```diff
@@ -19,16 +19,17 @@ To learn about the options for configuring the transition from one status to ano
 
 To create a new work package status click the green ***+ New Status*** icon. 
 
-![work package status](image-20200211142716298.png)
-
 A new window will open, where you will be able to specify the following:
 
 1. Name the new work package status.
-2. Define if the new work package status closes a work package (e.g. relevant when filtering for closed for packages), e.g. a work package status "rejected" will set a work package technically on closed and it will not appear in the default work package table with Open Work packages.
-3. Define if this status is set as default value when creating new work packages. BEWARE: If you decide to set the new status as default it will impact all work packages, existing and future ones. See more below.
-4. Check if this status sets a **work package in read-only mode**. This means no work package attributes can be changed except the status.
-5. Set a **color** for the work package status. The color appears in the status button of the work packages or in the work package table.
-6. Save the new status by clicking **Create**.
+2. The %&nbsp;Complete value in [Status-based progress calculation mode](../work-package-settings/).
+3. Define if the new work package status closes a work package (e.g. relevant when filtering for closed for packages), e.g. a work package status "rejected" will set a work package technically on closed and it will not appear in the default work package table with Open Work packages.
+4. Define if this status is set as default value when creating new work packages. BEWARE: If you decide to set the new status as default it will impact all work packages, existing and future ones. See more below.
+5. Check if this status sets a **work package in read-only mode**. This means no work package attributes can be changed except the status.
+6. Set a **color** for the work package status. The color appears in the status button of the work packages or in the work package table.
+7. Check **Exclude from calculation of totals in hierarchy** if you do not want work packages with this status to *not* be included in the calculation of sums in a hierarchy. This is useful for statuses like *closed* or *rejected*.
```

Comment:
This should be removed. "Exclude from calculation of totals in hierarchy" is not availabl for 14.0.

Reply:
Commented out.